.github/workflows/release.yml

```diff
@@ -73,6 +73,7 @@ jobs:
         uses: docker/build-push-action@ac9327eae2b366085ac7f6a2d02df8aa8ead720a
```

Comment:
For some reason, some actions use commit rather than using release version. I think should move the release version. wdyt?

Reply:
Absolutely!

Reply:
This is so that the builds are stable and not "upgrading" actions between runs. It more or less requires dependabot to make the updates though (which is a constant spam :/). Feel free to opt out of this and use tagged versions instead, it lowers the maintenance burden quite a bit.